pkg/ebpf/debugfs.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build linux_bpf
+//go:build !windows
```

Comment:
Why the change in build tag?

Reply:
as this is now being pulled in by `command.go` which is used everywhere, in CI there were cases on linux where `linux_bpf` is not set, so this definition fails to be found

Reply:
I think this file can stay the same, but have the real `NewDebugFsStatCollector` use `linux_bpf` and the no-op version use `!linux_bpf`. This reporter won't report anything if ebpf isn't enabled.

Reply:
yeah that's better, I'll change